agu_oss/clean_data.py

```diff
@@ -33,6 +33,6 @@ def open_and_clean(url):
     temp_df = temp_df.assign(Day=1)
 
     # Finally create a date time column
-    temp_df["date"] = pd.to_datetime(temp_df[['Year', 'Month', 'Day']])
+    temp_df["date"] = pd.to_dddddddddddddddatetime(temp_df[['Year', 'Month', 'Day']])
```

Comment:
@facusapienza21 did you really mean to misspell this ??

Reply:
That was exactly my intention :)

Reply:
nice work in breaking things @facusapienza21 ! :) 🚀

Reply:
I didn't know we could comment each other's pull requests!

Reply:
pretty, cool, eh @Cecile-A-C !? you can comment on any open  pull requests  which really makes collaboration and peer review effective on github!